gradle/tests.gradle

```diff
@@ -64,3 +64,15 @@ tasks.withType(Test) {
     }
 
 }
+
+allprojects {
```

Comment:
you dont need an extra section. update the one above `tasks.withType(Test) { ... }`